service/scaladsl/kinesis/client/src/main/scala/com/lightbend/lagom/scaladsl/broker/kinesis/KinesisMetadataKeys.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2018 Quotable Value Limited. <https://github.com/Quotable-Value>
```

Comment:
Stuart from Street Contxt here. We'd prefer if this copyright wasn't included -- would you be OK with removing? If so I'll merge this in (we're upgrading some internal libs to 2.12).

Apologies for the delay -- we got busy and this PR was forgotten